src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Extensions/PropertyExtensions.cs

```diff
@@ -39,6 +39,24 @@ public static IList<Property> ExpandByValues(this IEnumerable<Property> properti
             }).ToList();
         }
 
+        /// <summary>
+        /// Sorts properties by Priority attribute, then flattens the key-value tree
+        /// </summary>
+        public static IList<Property> ExpandOrderedByValues(this IEnumerable<Property> properties, string cultureName)
```

Comment:
maybe change to ExpandOrderedPropertiesByValues or ExpandAndOrderByValues?

Reply:
ExpandAndOrderByValues - sounds like "order by values" but it is actually ordering by specific properties
ExpandOrderedPropertiesByValues - the word "Properties" is redundant because the extension method is already intended for properties collection